docs/scos/dev/setup/installing-spryker-with-docker/quick-install-linux.md

```diff
@@ -45,35 +46,44 @@ git clone https://github.com/spryker/docker-sdk.git --single-branch docker
 
 ## 4. Bootstrap the local Docker environment
 
+1. Run the following command:
+
 ```shell
 docker/sdk bootstrap deploy.dev.yml
 ```
 
-{% info_block infoBox "Good to know" %}
+{% info_block infoBox "Info" %}
 
-If you do not need to build Spryker with development tools and file synchronisation, just run `docker/sdk bootstrap` instead. This mode is better suited to showcase Spryker features.
+If you do not need to build Spryker with development tools and file synchronization, run the following command:
 
-{% endinfo_block %}
+```shell
+docker/sdk bootstrap
+```
+This mode is better suited to showcase Spryker features.
 
-### 4.a Update the host file
+{% endinfo_block %}
 
-Update your `/etc/hosts` file with the command that the `docker/sdk bootstrap` command will display.
+2. Update the host file `/etc/hosts` file with the command that the `docker/sdk bootstrap` command displays.
 
 ## 5. Build and start your local instance
 
 ```shell
```

Comment:
@lenadoc For optional commands, we should not put it under shell. It breaks the UI